### PR TITLE
Match by regex, cleanup Dockerfile, listen on port 80 instead of 3000.

### PR DESCRIPTION
**Pull request by iakat**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,8 +28,8 @@ app.get('/:username/:repository.atom', function (req, res) {
   }
 });
 
-app.listen(3000, function () {
-  console.log('Docker RSS Feed lisening on port 3000!');
+app.listen(80, function () {
```
Comment (TheConnMan) [inline index.js:31]: I'd prefer to keep this on the traditional Express port. The Docker container can map the internal port to port 80 at runtime.

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,11 @@
-FROM alpine:3.4
+FROM node:8.1-alpine
```
Comment (TheConnMan) [inline Dockerfile:1]: Is getting to use Node 8 the reason you want to use the Node image? The Node image is twice as large as installing Node directly on Alpine.

File: index.js [hunk 3]
```diff
@@ -3,6 +3,8 @@ var RSS = require('rss');
 var express = require('express');
 var app = express();
 
+var regex_exclude = process.env.DOCKER_HUB_RSS_REGEX_EXCLUDE || false;
```
Comment (TheConnMan) [inline index.js:6]: Could this be passed in as a query param so it's dynamic? I see this overlapping with the include/exclude by name in the `dev` branch: https://github.com/TheConnMan/docker-hub-rss/blob/dev/public/js/app.js#L16.

**Review by TheConnMan — COMMENTED** [review]
Thanks for your PR! I've made some comments above.

File: Dockerfile (hunk 2 above)
Comment (iakat) [reply Dockerfile:1]: It feels more idempotent; node8 doesn't hurt

File: index.js (hunk 3 above)
Comment (iakat) [reply index.js:6]: I wasn't aware of that branch; feel free to close this MR in this case. Thanks! :D

Comment (TheConnMan) [reply index.js:6]: My fault, I'm always on the fence about which branch should be the default. I'll release the dev branch now so you can get the latest and greatest!

File: Dockerfile (hunk 2 above)
Comment (TheConnMan) [reply Dockerfile:1]: And while I'm at it I'll update to the Node version, I do want Node 8 and NPM 5.

**Comment by TheConnMan** [comment]
I'll close this PR, but thanks again for the contribution!

**Pull request closed without merging**